demos/ASP.Net MVC/Divergent.ITOps.ViewModelComposition/Divergent.ITOps.ViewModelComposition.csproj

```diff
@@ -109,6 +88,8 @@
     <Compile Include="Properties\AssemblyInfo.cs" />
     <Compile Include="RequestContext.cs" />
     <Compile Include="StringExtensions.cs" />
+    <Compile Include="Subscription %28generic%29.cs" />
```

Comment:
This file name feels a bit awkward, and it doesn't allow for the introduction of a separate file for `Subscription<T, V>`.

I'm not sure there's really an accepted standard for naming the file containing the generic equivalent of an existing non-generic type, but one I've seen used previously is ``Subscription`1.cs``, i.e. matching the "CLR type name".